packages/Chem/src/widgets/properties.ts

```diff
@@ -47,6 +52,30 @@ export function getChemPropertyFunc(name: string) : null | ((smiles: string) =>
   return null;
 }
 
+function getPropertyValue(molCol: DG.Column, idx: number, p: IChemProperty) {
```

Comment:
Function return types need to _always_ be annotated. The only exception is void.